src/Saritasa.NetForge.Tests/EfCoreDataServiceTests/DeleteEntityTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Saritasa.NetForge.Infrastructure.Abstractions.Interfaces;
+using Saritasa.NetForge.Tests.Domain;
+using Saritasa.NetForge.Tests.Domain.Models;
+using Saritasa.NetForge.Tests.Fixtures;
+using Saritasa.NetForge.Tests.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Microsoft.DependencyInjection.Abstracts;
+using Xunit.Microsoft.DependencyInjection.Attributes;
+
+namespace Saritasa.NetForge.Tests.EfCoreDataServiceTests;
+
+/// <summary>
+/// Delete entity tests.
+/// </summary>
+[TestCaseOrderer(Constants.OrdererTypeName, Constants.OrdererAssemblyName)]
+public class DeleteEntityTests : TestBed<NetForgeFixture>
+{
+#pragma warning disable CA2213
+    private readonly TestDbContext testDbContext;
+#pragma warning restore CA2213
+    private readonly IOrmDataService efCoreDataService;
+
+    /// <summary>
+    /// Constructor.
+    /// </summary>
+    public DeleteEntityTests(ITestOutputHelper testOutputHelper, NetForgeFixture netForgeFixture)
+        : base(testOutputHelper, netForgeFixture)
+    {
+        testDbContext = _fixture.GetService<TestDbContext>(_testOutputHelper)!;
+        efCoreDataService = _fixture.GetService<IOrmDataService>(_testOutputHelper)!;
+    }
+
+    /// <summary>
+    /// Delete valid entity test.
+    /// </summary>
+    [Fact]
+    [TestOrder(1)]
+    public async Task DeleteEntity_ValidEntity_Success()
+    {
+        // Arrange
+        var contactInfoType = typeof(ContactInfo);
+        var contactInfo = Fakers.ContactInfoFaker.Generate();
+
+        // Add the contactInfo to the database before attempting to delete it
+        await efCoreDataService.AddAsync(contactInfo, contactInfoType, CancellationToken.None);
```

Comment:
I think it's better to add directly to db context without `efCoreDataService` because this is more isolated way
```suggestion
        testDbContext.Add(contactInfo);
```

Reply:
Updated.